SECURITY_CONSIDERATIONS.md

```diff
@@ -0,0 +1,6 @@
+# Security Considerations
+
+OpenAPI documents use JSON, YAML and JSON Schema and therefore share their same security considerations. [JSON Schema Core](https://json-schema.org/draft/2020-12/json-schema-core#section-13) [JSON Schema Validation](https://json-schema.org/draft/2020-12/json-schema-validation#name-security-considerations) [YAML](https://www.ietf.org/archive/id/draft-ietf-httpapi-yaml-mediatypes-10.html) [JSON](https://www.rfc-editor.org/rfc/rfc8259)
+In additional, OpenAPI documents are processed by a wide variety of tooling for numerous different purposes, such as client code generation, documentation generation, server side routing, and API testing. OpenAPI document authors must consider the risks of the scenarios where the OpenAPI document may be used.
+OpenAPI documents may contain references to external resources that may be dereferenced automatically by consuming tools. References in an OpenAPI document, or across OpenAPI documents may cause a cycle. Tooling must detect and handle cycles to prevent resource exhaustion.
```

Comment:
Should there be something here about the security concerns of unrestricted nextwork access, e.g. following a chain of references off to some other authority domain that might host something malicious?  Or is this just considered too obvious to bother mentioning?

Reply:
I added some extra words to highlight the risk of accessing an untrusted domain.

Reply:
@darrelmiller thanks, this looks good!